### DISCUSSION TITLE
Create and Edit mygames.xml on PC

### DISCUSSION THREAD

**Discussion by KhenemetHeru** [aldostools/webMAN-MOD#722] [General]
Is there a way to create/edit the XML file on the server PC rather than by scanning on the PS3? 

I've had some bugginess when scanning my available games as it's a large list even without ISO formatted PSN games, and I would like to add them as well - I feel like I would run up against a memory error if I try - but as I understand it once the XML is generated that no longer applies, so I would like to create and organize that file on my PC if that's possible and upload it to my PS3 for Webman MOD to use.

**Comment by aldostools** [comment]
The easiest way to use a custom XML is:
1- Disable the content scanning at startup in /setup.ps3 to prevent that it replace the custom XML
2- Scan the content to generate an initial mygames.xml in /dev_hdd0/xmlhost/game_plugin
3- Download the generated /dev_hdd0/xmlhost/game_plugin/mygames.xml
4- Use [FormatXML](https://github.com/aldostools/webMAN-MOD/raw/master/_Projects_/tools/FormatXML.zip) to add indentation and amke the code easier to read.
5- Use a text editor with XML support like Notepad++or EditPlus.
6- Make your customization, save the file as mygames.xml and upload it to /dev_hdd0/xmlhost/game_plugin

Also you can use this template [wm_custom.xml](https://github.com/aldostools/webMAN-MOD/blob/master/_Projects_/updater/pkgfiles/USRDIR/xmb/wm_custom.xml) and upload it to /dev_hdd0/wm_custom.xml
